core/include/traccc/finding/finding_config.hpp

```diff
@@ -41,6 +45,10 @@ struct finding_config {
     /// Propagation configuration
     detray::propagation::config propagation{};
 
+    /// Particle hypothesis
+    detray::pdg_particle<traccc::scalar> ptc_hypothesis =
+        detray::muon<traccc::scalar>();
```

Comment:
Should the default hypothesis be a muon for the track finding? Would it not be better to have it as a pion? :thinking: